deploy.sh

```diff
@@ -47,4 +47,7 @@ eval "${KUBECTL} create configmap es-config --from-file=es-config --dry-run -o y
 eval "${KUBECTL} create configmap fluentd-config --from-file=docker/fluentd/td-agent.conf --dry-run -o yaml" | eval "${KUBECTL} apply -f -"
 eval "${KUBECTL} create configmap kibana-config --from-file=kibana.yml --dry-run -o yaml" | eval "${KUBECTL} apply -f -"
 
+## Install RBAC policies
+eval "${KUBECTL} apply -f rbac"
+
```

Comment:
Done. I've also added the serviceAccount directive to the manifests which I missed out earlier.

Btw, the RBAC manifests were meant for ES2. ES5 is missing the `k8s-events-printer.yaml` and `es-fluentd-ds.yaml` manifests present in ES2. Is the ES5 deployment ready for use? I tried it a few weeks back but it didn't run properly.